client/apiClient.js

```diff
@@ -50,6 +50,10 @@ export function addOrderItem (userId, orderId) {
     .send(data)
 }
 
+export function deleteItem (userItemId) {
+  return request.delete(`/api/v1/current-order/${userItemId}`)
```

Comment:
Let's rename the parameter to `orderItemId` because I don't think we're using `userItemId` anywhere else.